tools/webassembly.py

```diff
@@ -110,7 +117,6 @@ def add_emscripten_metadata(wasm_file):
 
 def parse_dylink_section(wasm_file):
   wasm = open(wasm_file, 'rb').read()
-  section_name = b'\06dylink' # section name, including prefixed size
```

Comment:
looks like we lost an assertion on the section name in the refactor? do you think it's not needed?

Reply:
Thanks!  Done